newrelic-alerts-configurator/src/main/java/com/ocadotechnology/newrelic/alertsconfigurator/configuration/condition/nrql/NrqlUtils.java

```diff
@@ -0,0 +1,19 @@
+package com.ocadotechnology.newrelic.alertsconfigurator.configuration.condition.nrql;
+
+import com.ocadotechnology.newrelic.apiclient.model.conditions.nrql.Nrql;
+
+public final class NrqlUtils {
+
+    private NrqlUtils() {
+    }
+
+    public static Nrql createNrql(NrqlConfiguration nrqlConfiguration) {
+        if(nrqlConfiguration != null) {
```

Comment:
`nrql` is annotated as `@NotNull` in `NrqlCondition`. It should never be null here.